src/utils/work/fetchWork.js

```diff
@@ -68,11 +68,6 @@ const fetchWork = async (
     throw new Error('Disabling network interaction on server');
   }
 
-  const { environment } = getState().networkResources;
-  if (environment === Environment.DEVELOPMENT && !localStorage.getItem('disableCache')) {
-    localStorage.setItem('disableCache', 'true');
-  }
-
```

Comment:
Removing these lines means that browser cache in the UI will always be enabled. Why are we making this change? This caching is different from the caching that is getting disabled in non-production environments in the API here: https://github.com/hms-dbmi-cellenics/api/pull/511

Reply:
is that true? from my understanding, this `localStorage.setItem('disableCache', 'true');` only stores a `disableCache` flag that is used to decide whether to send the `cacheUniquenessKey` to the API. No relation to the local storage in the browser.

Reply:
What you are saying is true, but because the ETag was generated from the UI, this was effectively disabling the cache in local storage, hence the relation with local storage. Now that the ETag is computed from the API and hence for every request we reach to the API to get the ETag, it is true that the local storage cache of the UI can be disabled from the API. I find it weird, because the browser cache feels like something specific to the UI, hence it should be controlled by the UI. But I think it is ok as is, your changes as they are are good, I will approve shortly